templates/DatasetMonitoring/index.html.twig

```diff
@@ -8,9 +8,16 @@
     <link href="/css/dx.generic.griidc-scheme.css" rel="stylesheet">
 
     <style>
+      .dx-widget {
+        font-size: 1rem!important;
+        font-family: dm-sans, Helvetica, sans-serif!important;
```

Comment:
I feel we should remove this, because the font is set by the template, and it's set to this.